MatrixSDK/Data/MXRoomSummaryUpdater.m

```diff
@@ -170,9 +203,19 @@ - (BOOL)session:(MXSession *)session updateRoomSummary:(MXRoomSummary *)summary
             }
                 
             case MXEventTypeRoomCreate:
+            {
+                MXRoomCreateContent *createContent = [MXRoomCreateContent modelFromJSON:event.content];
                 summary.creatorUserId = roomState.creatorUserId;
-                updated = YES;                
-                [self checkRoomCreateStateEventPredecessorAndUpdateObsoleteRoomSummaryIfNeededWithCreateEvent:event summary:summary session:session roomState:roomState];
+                                
+                NSString *roomTypeString = [createContent.roomType stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceAndNewlineCharacterSet]];
```

Comment:
It is really required?
I mean do we need to accept bad formed type?

Reply:
Just in case someone made a bad blank room type through dev tools